element_calcium_imaging/imaging_report.py

```diff
@@ -0,0 +1,414 @@
+import cv2
+import numpy as np
+import datajoint as dj
+from functools import partial
+from plotly.io import from_json
+import plotly.graph_objects as go
+from ipywidgets import widgets as wg
+from . import imaging, scan
```

Comment:
Unfortunately, this cannot be quite as straightforward. We don't know which `imaging` to import (e.g. is it the `-no-curation` version, or the `-preprocess`, or whichever else may exist in the future). This is all dependent on which one being imported and activated at the workflow level, which is essentially unknown as this stage.

The far easier solution is to move all of this code to the "workflow" level, but in my view it is a far inferior solution, just easier to implement.

This plotting feature should come with every workflow using the `element-calcium-imaging`. So, toward solving this chicken-and-egg problem, Jaerong and I proposed a solution - see https://github.com/JaerongA/element-array-ephys/blob/plotly/element_array_ephys/ephys_report.py

This solution is not exactly elegant (yet), but it is generalizable and should work. We will further optimize this

Reply:
@kabilar the recent updates address this issue - recommending to resolve this comment

Reply:
Thanks @ttngu207!  The method you proposed is great.